internal/stack/serverless.go

```diff
@@ -273,10 +290,19 @@ func (sp *serverlessProvider) BootUp(options Options) error {
 		printUserConfig(options.Printer, config)
 	}
 
-	logger.Infof("Starting local agent")
-	err = sp.startLocalAgent(options, config)
+	logger.Infof("Starting local services")
+	err = sp.startLocalServices(options, config)
 	if err != nil {
-		return fmt.Errorf("failed to start local agent: %w", err)
+		return fmt.Errorf("failed to start local services: %w", err)
+	}
+
+	// Updating the output with ssl certificates created in startLocalServices
+	// The certificates are updated only when a new project is created and logstash is enabled
+	if isNewProject && settings.LogstashEnabled {
+		err = project.UpdateLogstashFleetOutput(sp.profile, sp.kibanaClient)
+		if err != nil {
+			return err
+		}
 	}
```

Comment:
Do we need this if we are doing it only on new projects? We are already creating the output in https://github.com/elastic/elastic-package/pull/1646/files#diff-b4efb1ac558d15d3077525cb9be59dce12ee723a4749253e9ca7cc860419f1f8R123

Reply:
During the `Add` the certificates are not yet created. Because they are created after the project is created and `startLocalServices` is called. Hence the agent certificates are to be updated in the output created.